editorconfig-textmate.xcodeproj/project.pbxproj

```diff
@@ -484,7 +484,8 @@
 					"$(PROJECT_DIR)/lib",
 					"$(PROJECT_DIR)/lib/pcre",
 				);
-				ONLY_ACTIVE_ARCH = NO;
+				MARKETING_VERSION = 0.4.1;
+				ONLY_ACTIVE_ARCH = YES;
```

Comment:
MacOS SDKs now support Apple M1 by default, but this only includes an x86_64 version of libpcre2. I need to figure out how to dual-build *that* before I can build more architectures here.